QuipEditor.py

```diff
@@ -171,6 +172,21 @@ def on_pre_close(self, view):
 		manager.remove_tab(view=view)
 
 
+class ShowCommentsOnHover(sublime_plugin.EventListener):
+
+	def on_hover(self, view, point, hover_zone):
```

Comment:
It works too strange,  carriage follow the pointer in any position of opened document

Reply:
You have clicked and hold the mouse button
And at the same moment, while its was in selecting mode, you hovered on the id section and hold the mouse on it for a second so the `on_hover` method is triggered
And so, probably because of `view.sel().add(Region(point))` it was following for your selection
At least, it was my case to reproduce this bug. If I use sublime as usual, it was ok.

I add this, because the `show_popup_menu` function show menu on the line where your carriage are (and it looks better, than just raw popup). And we can add a callback on the click
![image](https://user-images.githubusercontent.com/73075768/102761887-989d7000-4388-11eb-9789-674ab2bceb4e.png)

And seems like we cannot change the popup height and it looks ugly:

![image](https://user-images.githubusercontent.com/73075768/102762190-ff228e00-4388-11eb-9357-94f85d1a015f.png)